crypto/txsigner_london.go

```diff
@@ -0,0 +1,77 @@
+package crypto
+
+import (
+	"crypto/ecdsa"
+	"math/big"
+
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+// LondonSigner implements signer for EIP-1559
+type LondonSigner struct {
+	chainID        uint64
+	fallbackSigner TxSigner
+}
+
+// NewLondonSigner returns a new LondonSigner object
+func NewLondonSigner(chainID uint64, fallbackSigner TxSigner) *LondonSigner {
```

Comment:
Why do we need a fallback signer?

Reply:
We use london signer if a london hardfork is enabled. Even with enabled london hardfork, users could still send legacy transactions that should be handled by fallback signer.

Reply:
Also, I didn't want to duplicate the code by moving the legacy tx encoding to the london signer.